parsec/core/gui/workspace_roles.py

```diff
@@ -0,0 +1,18 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+from parsec.core.types import WorkspaceRole
+from parsec.core.gui.lang import translate as _
+
+
+NOT_SHARED_KEY = "NOT_SHARED"
+
+
+def get_role_translation(user_role):
+    ROLES_TRANSLATIONS = {
```

Comment:
I did it this way because I remembered having problems with translations defined globally (probably something to do with the order it gets interpreted, the translations are performed before we actually load the file and select the language).